tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Debugger/snapshots/ProbesTests.MethodProbeTest_testType=Samples.Probes.SmokeTests.AsyncCallChain.verified.txt

```diff
@@ -0,0 +1,191 @@
+﻿[
+  {
+    "dd.span_id": "ScrubbedValue",
+    "dd.trace_id": "ScrubbedValue",
+    "ddsource": "dd_debugger",
+    "ddtags": "Unknown",
+    "debugger": {
+      "snapshot": {
+        "captures": {
+          "entry": {
+            "arguments": {
+              "chain": {
+                "fields": null,
+                "type": "Int32",
+                "value": "1"
+              }
+            },
+            "fields": {
+              "_chain": {
+                "fields": null,
+                "type": "Int32",
+                "value": "1"
+              }
+            }
+          },
+          "return": {
+            "arguments": {
+              "chain": {
+                "fields": null,
+                "type": "Int32",
+                "value": "2"
+              }
+            },
+            "fields": {
+              "_chain": {
+                "fields": null,
+                "type": "Int32",
+                "value": "1"
+              }
+            },
+            "locals": {
+              "@return": {
+                "fields": null,
+                "type": "Int32",
+                "value": "3"
+              },
+              "result": {
+                "fields": null,
+                "type": "Int32",
+                "value": "3"
+              }
+            }
+          }
+        },
+        "duration": "ScrubbedValue",
+        "id": "ScrubbedValue",
+        "language": "dotnet",
+        "probe": {
+          "id": "ScrubbedValue",
+          "location": {
+            "method": "MoveNext",
```

Comment:
We definitely don't have to handle it in this PR, but the method name here should be fixed to denote the logical method name.

Reply:
Fixed in 82878f89777debace4f5f6ff669306f87e4c1ae7